conf.py

```diff
@@ -372,3 +373,18 @@
 
 # If false, no index is generated.
 #epub_use_index = True
+
+
+redirect_files = ['creating_packages/package_dev_flow.html']
```

Comment:
Could make sense to covert it to a dict of {origin: dest} and autogenerating the redirect html?

Reply:
Done